src/main/java/services/event_creation/EventAdder.java

```diff
@@ -12,7 +20,18 @@ public EventAdder(CalendarManager calendarManager) {
 
     @Override
     public void addEvent(CalendarEventModel eventData) {
-        calendarManager.addEvent(eventData);
+
+        String eventName = eventData.getName();
+        HashSet<String> tags = eventData.getTags();
+        DatesForm form = eventData.getForm();
+        Duration eventDuration = eventData.getDuration();
+
+        StrategyBuilderDirector director = new StrategyBuilderDirector();
+        DateStrategy strategy = director.createStrategy(form);
+        List<LocalDateTime> times = strategy.datesBetween(LocalDateTime.now(), LocalDateTime.now().plusYears(1));
```

Comment:
does this list of times ever get used again?

Reply:
Like is there currently a way to view all scheduled events from now until x date

Reply:
oh nevermind I just reread your initial comment on this PR